apps/system/init/init.c

```diff
@@ -69,6 +69,11 @@
 extern void iotjs_register_cmds(void);
 #endif
 
+#ifdef CONFIG_BUILD_FLAT
```

Comment:
```

The vwifi_start symbol is defined in kernel. So, when loadable_apps config
is used, this symbol cannot be accessed directly from app binary, and needs
to be accessed via syscall or ioctl. As a temporary fix, we check for
CONFIG_BUILD_FLAT in the framework code to avoid linker error.
```
There is no reason why you moved this from framework to preapp. Could you let me know and write it on commit description?

Reply:
Pilseob made PR #4908 for this. Let's remove this commit from this PR.

Reply:
Okay. Done.